File: more/pointer/2.py
#Find all pairs whose sum equals a given target.
a = list(map(int, input("Enter a list: ").split()))
target = int(input("Enter a target: "))
left = 0
right = len(a)-1
while left<right:
    total = a[left]+a[right]
    if total == target:
        print(a[left],a[right])
    elif total<target:
        left+=1
    else:
        right-=1

File: extra/random/2.c
int a[] = {2, 4, 6, 8, 10};
int sum = 0;

for(int i = 0; i < 5; i++) {
    if(a[i] % 4 == 0)
        sum += a[i];
    else
        sum -= a[i];
}

printf("%d", sum);